payersAndReceivers.md

```diff
@@ -0,0 +1,27 @@
+# Yield Swap
+
+![Payer Profit](image/payers/payer_profit.png)
+
+## Payers and Receivers
+
+The yield swap is a bilateral agreement between a Payer and a Receiver with the following terms:
+
+1. notional value (say USD 1000),
+1. tenor (say 10 days), and
+1. fixed interest rate (say 5% p.a.).
+
+At maturity, the Payer pays the fixed-interest rate (applied to the notional for the tenor) to the Receiver, and in return receives the actual realized floating yield (applied to the notional for the tenor) from the Receiver.
+
+Put differently, yield swap is a [Future-Contract] where the Payer is said to "buy" or go "long" the floating yields, and the Receiver is said to "sell" or go "short" the floating yield.
+
+Whereas Payers bet on higher floating yields, Receivers bet on the opposite.
+
+## Net Settlement
+
+Only the net amount is settled at maturity, i.e., the difference between the realized floating and fixed interest rate (applied to the notional during the tenor).
+
+## Profit and Loss
+
+In case the floating yield realized higher than the fixed yield, the Payer makes a profit (net is paid by the Receiver), and otherwise the Receiver makes a profit (net is paid by the Giver).
```

Comment:
I ncase the realized (accumulated) floating yield is higher than the fixed yield , ..

Reply:
changed